src/index.ts

```diff
@@ -1,28 +1,57 @@
-import { ApolloServer } from '@apollo/server'
-import { startStandaloneServer } from '@apollo/server/standalone'
+import 'reflect-metadata';
+import { ApolloServer, gql } from 'apollo-server';
+import { AppDataSource } from './data-source';
+import { User } from '../entities/User';
 
-const typeDefs = `
+const typeDefs = gql`
+  type User {
+    id: ID
+    name: String
+  }
 
-    type Query {
-        hello: String
-    }
+  type Mutation {
+    createUser(name: String!): User
```

Comment:
qual a diferença de colocar ou não o `!`
```suggestion
    createUser(name: String!): User!
```

Reply:
pra dize que o parâmetro name é obrigatório